packages/web-components/fast-foundation/src/horizontal-scroll/horizontal-scroll.template.ts

```diff
@@ -10,6 +10,24 @@ export const HorizontalScrollTemplate: ViewTemplate<HorizontalScroll> = html`
     <template class="horizontal-scroll">
         ${startTemplate}
         <div class="scroll-area">
+            ${when(
+                x => x.view !== "mobile",
+                html<HorizontalScroll>`
+                    <div
+                        class="scroll scroll-prev"
+                        part="scroll-prev"
+                        ${ref("previousFlipper")}
+                    >
+                        <button
+                            class="scroll-action"
+                            @click="${x => x.scrollToPrevious()}"
+                            tabindex="-1"
+                        >
```

Comment:
If I slot a flipper we have buttons inside buttons essentially - I think we discussed potentially doing this in the feature branch to leverage the new API for default slotted content. I think that makes this story a lot easier.

Reply:
@chrisdholt  I wasn't sure if I should wait until vNext was merged back into master.